### PR TITLE
fix: adjust UiFormFieldCharacterCounter color to the design

### PR DESCRIPTION
**Pull request by AlanLes**
image

### CODE REVIEW

**Review by mateuszkulpa — DISMISSED** [review]
LGTM 🚀 👏

File: src/components/molecules/UiFormField/UiFormField.vue [hunk 1]
```diff
@@ -216,6 +216,10 @@ const handleErrorEmit = (error: string | null) => {
 
   &__character-counter {
     margin-inline: auto 0;
+
+    &--has-error {
```
Comment (pspaczek) [inline src/components/molecules/UiFormField/UiFormField.vue:220]: Q: why did you add a class for UiFormFieldCharacterCounter to UiFormField? IMO it's a part of UiFormFieldCharacterCounter. You use it in it.

Comment (AlanLes) [reply src/components/molecules/UiFormField/UiFormField.vue:220]: It was added here in the first place, I just followed this pattern: the `__character-counter`  was added here while it's used in the child component. Am I wrong?

File: src/components/molecules/UiFormField/_internal/UiFormFieldCharacterCounter.vue [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <template>
   <UiText
     class="ui-text--body-2-comfortable ui-text--theme-secondary ui-form-field__character-counter"
+    :class="{ 'ui-form-field__character-counter--has-error': length > max }"
```
[the remark this answers is not in the document]
Comment (AlanLes) [reply src/components/molecules/UiFormField/_internal/UiFormFieldCharacterCounter.vue]: OK, it's done

File: src/components/molecules/UiFormField/UiFormField.vue (hunk 1 above)
Comment (pspaczek) [reply src/components/molecules/UiFormField/UiFormField.vue:220]: Yes you are wrong 🙈; It'a a part of character counter; in parent, you should use "positioning properties" like margins, padding etc.
